proto/prysm/v1alpha1/blobs.proto

```diff
@@ -37,3 +37,24 @@ message BlobsSidecar {
         // The aggregated proof for blobs and commitments for validation.
         bytes aggregated_proof = 4 [(ethereum.eth.ext.ssz_size) = "48"];
 }
+
+message BlobSidecar {
+        bytes block_root = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+        uint64 index = 2;
+        uint64 slot = 3 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives.Slot"];
+        bytes block_parent_root = 4 [(ethereum.eth.ext.ssz_size) = "32"];
+        uint64 proposer_index = 5 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives.ValidatorIndex"];
+        ethereum.engine.v1.Blob blob = 6;
+        bytes kzg_commitment = 7 [(ethereum.eth.ext.ssz_size) = "48"];
```

Comment:
wonder if there would be some benefit to adding primitives for` kzg_commitment` and `kzg_proof`.